docker-compose.yml

```diff
@@ -13,16 +13,13 @@ services:
 
   firefly-app:
     image: firefly-iii
-    build:
-      context: .
+    build: .
     environment:
-      - INIT_DATABASE=yes
```

Comment:
I would keep this here with a comment that you need to set it to "no" after the first init. You still need to run it with "yes" the first time you run the production config.

Or even better, we can use this aproach [here](https://docs.docker.com/compose/extends/#/administrative-tasks) to run tasks like _init-database_ or _compile-assets_ (gulp).